### PR TITLE
manifest: Update hostap to fix EAP server registration

### PR DESCRIPTION
**Pull request by MaochenWang1**
Fix the duplicated registration of eap server method, as when hostapd enabled, eap_server_register_methods() will also call the eap server related functions.

### CODE REVIEW

**Comment by jukkar** [comment]
@MaochenWang1 the hostap PR is now merged, please update the sha in west.yml and resubmit

**Comment by jukkar** [comment]
No need to close this pr, you can fix the commit and force push a new version

**Comment by MaochenWang1** [comment]
> No need to close this pr, you can fix the commit and force push a new version

Hi @jukkar current status shows closed
image
you mean this is not enough? how to fix the commit and force push a new version?

**Comment by jukkar** [comment]
There should be a reopen button below, I just pressed it.

**Comment by MaochenWang1** [comment]
> fix the commit and force push a new version

Hi @jukkar , I still don't understand what's your meaning of fix the commit, can you explain more about it? thanks

File: west.yml [hunk 1]
```diff
@@ -259,7 +259,7 @@ manifest:
         - hal
     - name: hostap
       path: modules/lib/hostap
-      revision: a90df86d7c596a5367ff70c2b50c7f599e6636f3
+      revision: pull/23/head
```
Comment (jukkar) [inline west.yml:262]: You need to fix the revision and put here the correct SHA from hostap module that has your change.

Comment (MaochenWang1) [reply west.yml:262]: yes, but I already in another PR update the latest SHA of hostap:
https://github.com/zephyrproject-rtos/zephyr/pull/76824
so I wanted to close this PR as it's not needed anymore,  am I wrong?

Comment (jukkar) [reply west.yml:262]: Sure, if the manifest is already updated, this can be closed.

**Comment by jukkar** [comment]
Manifest already updated so closing.

**Pull request closed without merging**